modal/volume.py

```diff
@@ -341,5 +341,17 @@ async def _upload_file(
             mode=file_spec.mode,
         )
 
+    @live_method
+    async def copy_files(self, src_paths: Sequence[Union[str, bytes]], dst_path: Union[str, bytes]) -> None:
+        """
+        Copy files within the volume from src_paths to dst_path.
+        """
+        src_paths = [path.encode("utf-8") for path in src_paths if isinstance(path, str)]
+        if isinstance(dst_path, str):
+            dst_path = dst_path.encode("utf-8")
+
+        request = api_pb2.VolumeCopyFilesRequest(volume_id=self.object_id, src_paths=src_paths, dst_path=dst_path)
+        await retry_transient_errors(self._client.stub.VolumeCopyFiles, request, base_delay=1)
```

Comment:
Just thought of the idempotency of this RPC. Pretty sure it is idempotent, but we don't have a backend test for that property.